producers/src/end-to-end-test/resources/application-end-to-end-test.yaml

```diff
@@ -40,6 +40,8 @@ sdt:
   purge:
     cron: ${PURGE_SCHEDULE_CRON:0 10 5 * * ?}
     commitInterval: ${PURGE_COMMIT_INTERVAL:1000}
+  requeue:
+    cron: ${REQUEUE_SCHEDULE_CRON:0 20 0 * * ?}
```

Comment:
The default should be `0 */20 * * ?`. This means "Run every 20th minute" whereas `0 20 0 * * ?` means "Run at 00:20:00"

Reply:
done